README.md

```diff
@@ -1,61 +1,27 @@
 React Firebase
 ==============
 
-React bindings for [Firebase](https://www.firebase.com).
+React bindings for [Firebase](https://firebase.google.com).
 
 ## Installation
 
-React Firebase requires **React 0.14 or later.**
+React Firebase requires **React 0.14 and Firebase 3 or later.**
 
 ```
 npm install --save react-firebase
 ```
 
 ## Usage
```

Comment:
A simple example should give you a glance of what you could do with it. I Often start using a lib before reading a docs if the example is short and good enough.

```js
import firebase from 'firebase'
import { connect } from 'react-firebase'

firebase.initializeApp({
  databaseURL: 'https://react-firebase-sandbox.firebaseio.com'
})

const Counter = ({ value, setValue }) => (
  <div>
    <button onClick={() => setCount(value - 1)}>-</button>
    <span>{value}</span>
    <button onClick={() => setCount(value + 1)}>+</button>
  </div>
)

export default connect((props, ref) => ({
  value: 'counterValue',
  setValue: value => ref('counterValue').set(value)
}))(TodosApp)
```